airbyte-commons/src/main/java/io/airbyte/commons/exceptions/ConnectionErrorException.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
  */
 
-package io.airbyte.db.exception;
+package io.airbyte.commons.exceptions;
```

Comment:
For context, this class was moved to the `commons` package so that it can be utilized in the `SshTunnel` class and with the `Db` classes